### DISCUSSION TITLE
How to migrate to IPV6

### DISCUSSION THREAD

**Discussion by idofarit** [supabase/supabase#20920] [Questions]
I have cerated a react js application with the supabase database. Do I need to migrate to IPV6? If I dont then is there any problem for my app? Please help!!

**Comment by markitics** [comment]
If you just created the app, then you should be OK. If you're referring to an old app (where you set up the supabase db integration a while ago), see https://twitter.com/supabase/status/1749827349128913346

I had an old (Django) project with a manual (URL) connection, so I needed to make manual changes.
In settings.py, I updated my `USER` from `postgres` to `postgres.[mydatabaseid]` and update my `HOST` from `db.[mydatabaseid].supabase.co` to `aws-0-us-east-1.pooler.supabase.com`. That's all I had to do for a python app connecting using `psycopg2`.

If you've created a new project and followed the instructions on your https://supabase.com/dashboard/project/[yourprojectid]/settings/database page, then you're probably already OK with no further changes required.

Timelines per https://github.com/orgs/supabase/discussions/17817 
> PgBouncer will be available to use alongside Supavisor until **January 31st 2024.**
> 29th January 2024: Your Supabase database domain (db.projectref.supabase.co) will start resolving to IPv6 addresses. PgBouncer will be removed. Projects will be migrated over starting this day. No changes are required if your network supports communicating via IPv6. If it doesn't, update your applications to use Supavisor which will continue to return IPv4 addresses.
